docs/source/index.rst

```diff
@@ -1,6 +1,10 @@
+====
 Dask
 ====
 
+Dask  is a simple task scheduling system that breaks up large computations into many small ones.  
+The word "dask" comes from a combination of "DAG" (for directed acyclic graph) and "task." 
```

Comment:
I recommend omitting the origin of the name.

On Thu, Jan 7, 2016 at 2:29 PM, Will Warner <notifications@github.com>
wrote:

> In docs/source/index.rst
> <https://github.com/blaze/dask/pull/908#discussion_r49135634>:
>
> >  Dask
> >  ====
> >
> > +Dask  is a simple task scheduling system that breaks up large computations into many small ones.
> > +The word "dask" comes from a combination of "DAG" (for directed acyclic graph) and "task."
>
> Hm. I'll reword that and see what you think of the next version...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blaze/dask/pull/908/files#r49135634>.
>